modules/globebrowsing/src/tileprovider/tileproviderbyindex.cpp

```diff
@@ -67,21 +85,30 @@ documentation::Documentation TileProviderByIndex::Documentation() {
 TileProviderByIndex::TileProviderByIndex(const ghoul::Dictionary& dictionary) {
     ZoneScoped;
 
-    const Parameters p = codegen::bake<Parameters>(dictionary);
+    Parameters p = codegen::bake<Parameters>(dictionary);
+
+    // For now we need to impute the LayerGroupID this way. We don't want it to be part of
+    // the parameters struct as that would mean it would be visible to the end user, which
+    // we don't want since this value just comes from whoever creates us, not the user
```

Comment:
```suggestion
    // we don't want since this value just comes from whoever creates it, not the user
```

Is that what you mean?